cypress/integration/tests/developer/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -220,4 +220,22 @@ describe("Source Analysis", { tags: "@tier1" }, () => {
         application.openreport();
         application.validateStoryPoints();
     });
+
+    it("Analysis for known Open Source libraries on tackleTest app", function () {
+        // Source code analysis require both source and maven credentials
+        const application = new Analysis(
+            getRandomApplicationData("tackleTestApp_Source+dependencies", {
```

Comment:
**chore:** There is already an application created by this test suite called `tackleTestApp_Source+dependencies` (line 121) so this is probably going to fail because there can't be two apps with the same name.

Maybe rename it to `tackleTestApp_Source+knownLibraries` ?

Reply:
Just this comment to be implemented @kpunwatk .

Although the name appends a random string so it will be unique but let's give a name related to the test to identify the test . It helps in debugging .

Reply:
Thanks a lot for the suggestions @abrugaro @sshveta added the changes.